lib/plugins/aws/package/compile/events/apiGateway/lib/restApi.js

```diff
@@ -4,6 +4,22 @@ const _ = require('lodash');
 const BbPromise = require('bluebird');
 
 module.exports = {
+  resourcePolicy: {
```

Comment:
Let's not expose it on `module.exports` but keep as local variable

Reply:
@medikoo I exposed it on `module.exports` to easily make it available to the test and also to reference locally.  So make it local and have a duplicate policy in the test?